### PR TITLE
Implement mail notification on successful purchase.

### LINKED ISSUE #15: Send email after pruchase.

### PR DESCRIPTION
**Pull request by boxdot**
Closes #15.

### CODE REVIEW

File: tickets/models.py [hunk 1]
```diff
@@ -24,31 +25,56 @@ def create_ticket(self):
...
         ticket_count = len(self.tickets)
-        return "Your purchase of {} tickets for METZ".format(ticket_count)
+        return "Your purchase of {} tickets for {}".format(
+            ticket_count, self.event.title)
+
+    @property
+    def subject(self):
+        ticket_count = len(self.tickets)
+        if ticket_count == 1:
+            return "Your 1 ticket for {}".format(self.event.title)
```
Comment (jeschkies) [inline tickets/models.py:50]: Nit: "Your ticket for"

**Review by jeschkies — CHANGES_REQUESTED** [review]
Please document `POSTMARK_API_TOKEN` in README 😉.

**Comment by jeschkies** [comment]
> Coverage decreased (-2.7%) to 91.333% when pulling 393363b on notify into 0470a85 on master.

It should be safe to ignore `tickets/app.py` in `.coveragerc`.

File: tickets/models.py (hunk 1 above)
Comment (boxdot) [reply tickets/models.py:50]: Done.

**Review by jeschkies — APPROVED** [review]

**Pull request merged**